SLHCUpgradeSimulations/Configuration/python/aging.py

```diff
@@ -289,4 +289,3 @@ def ecal_complete_aging_3000(process):
                                     )
 
     return process
-
```

Comment:
better drop this empty line change so that the file is not on the list of modifications in this PR